central/reports/scheduler/schedule_test.go

```diff
@@ -0,0 +1,99 @@
+	"github.com/stackrox/rox/pkg/branding"
+	"github.com/stackrox/rox/pkg/fixtures"
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/assert"
+)
+
+var expectedVulnReportEmailTemplateRhacsBrandingWithPlaceholders = `
+	Red Hat Advanced Cluster Security for Kubernetes has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
+
+	To address these findings, please review the impacted software packages in the container images running within deployments you are responsible for and update them to a version containing the fix, if one is available.`
+
+var expectedVulnReportEmailTemplateStackroxBrandingWithPlaceholders = `
+	StackRox has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
+
+	To address these findings, please review the impacted software packages in the container images running within deployments you are responsible for and update them to a version containing the fix, if one is available.`
+
+var expectedNoVulnsFoundEmailTemplateRhacsBranding = `
+	Red Hat Advanced Cluster Security for Kubernetes has found zero vulnerabilities associated with the running container images owned by your organization.`
+
+var expectedNoVulnsFoundEmailTemplateStackroxBranding = `
+	StackRox has found zero vulnerabilities associated with the running container images owned by your organization.`
+
+type vulnsAndDate struct {
+	WhichVulns string
+	DateStr    string
+}
+
+func generateExpectedVulnReportEmailTemplates(t *testing.T) (string, string) {
+	data := &vulnsAndDate{
+		WhichVulns: "for all vulnerabilities",
+		DateStr:    time.Now().Format("January 02, 2006"),
+	}
+
+	tmpl, err := template.New("VulnsRHACS").Parse(expectedVulnReportEmailTemplateRhacsBrandingWithPlaceholders)
+	assert.Nil(t, err)
+	var tpl0 bytes.Buffer
+	err = tmpl.Execute(&tpl0, data)
+	assert.Nil(t, err)
+
+	expectedVulnReportEmailTemplateRhacsBranding := tpl0.String()
+
+	tmpl, err = template.New("VulnsStackrox").Parse(expectedVulnReportEmailTemplateStackroxBrandingWithPlaceholders)
+	assert.Nil(t, err)
+	var tpl1 bytes.Buffer
+	err = tmpl.Execute(&tpl1, data)
+	assert.Nil(t, err)
+
+	expectedVulnReportEmailTemplateStackroxBranding := tpl1.String()
+
+	return expectedVulnReportEmailTemplateRhacsBranding, expectedVulnReportEmailTemplateStackroxBranding
+}
+
+func TestVulnMessageBranding1(t *testing.T) {
+	envIsolator := envisolator.NewEnvIsolator(t)
+	rc := fixtures.GetValidReportConfiguration()
+
+	expectedVulnReportEmailTemplateRhacsBranding, expectedVulnReportEmailTemplateStackroxBranding := generateExpectedVulnReportEmailTemplates(t)
+
+	tests := []struct {
+		name            string
+		productBranding string
+		vulnReport      string
+		noVulnReport    string
+	}{
+		{
+			name:            "RHACS branding",
```

Comment:
If there's a test name, please make it into a map. It makes it easier to read.
```go
tests := map[string]struct{
   ...
}{
   "Test name": { ... }
}

// You can iterator on (key, value) using golang's range
for name, tt := range test {
   ...
}
```

Reply:
Done